deployment/bicep/vnettest/vnetWithBastian.bicep

```diff
@@ -0,0 +1,73 @@
+param namePrefix string = 'unique'
```

Comment:
File name probably contains spelling error "vnetWithBastiAn" instead of "vnetWithBastiOn"  <- capitals to emphasize